pythran/config.py

```diff
@@ -136,9 +135,6 @@ def fmt_define(define):
         output.append('-L' + distutils.sysconfig.get_config_var('LIBPL'))
         output.extend(('-l' + include)
                       for include in extension['libraries'])
-        output.extend(distutils.sysconfig.get_config_var('LIBS').split())
-        output.append('-lpython' +
-                      distutils.sysconfig.get_config_var('VERSION'))
```

Comment:
On Wed, Sep 02, 2015 at 11:57:43AM -0700, pbrunet wrote:
> > @@ -136,9 +135,6 @@ def fmt_define(define):
> >          output.append('-L' + distutils.sysconfig.get_config_var('LIBPL'))
> >          output.extend(('-l' + include)
> >                        for include in extension['libraries'])
> > -        output.extend(distutils.sysconfig.get_config_var('LIBS').split())
> > -        output.append('-lpython' +
> > -                      distutils.sysconfig.get_config_var('VERSION'))
> 
> You remove it here but you didn't add it in the default compilation. It won't compile no?

This code is only relevant to pythran-config. So any user of
pythran-config (=empty set) would need extra flags, but that's to cope
with boost python 2/3 difference